packages/cna-template/template/components/Meta/Meta.tsx

```diff
@@ -59,14 +54,6 @@ export const Meta: FC<PropsWithChildren<MetaProps>> = ({
       {image && <meta property="og:image:alt" content={image.alt} />}
       {updatedAt && <meta property="og:updated" content={updatedAt} />}
 
-      {/* Twitter */}
-      {twitter && <meta name="twitter:site" content={`@${twitter}`} />}
-      {image && <meta name="twitter:image:alt" content={image.alt} />}
-      <meta
-        name="twitter:card"
-        content={image ? 'summary_large_image' : 'summary'}
-      />
-
```

Comment:
What is the reason behind this removal?

Reply:
I refuse to support Twitter in any capacity at this point.

Reply:
Fair point.